### PR TITLE
Added contribute information to readme

### PR DESCRIPTION
**Pull request by story645**
Added a small contribute section to the bottom of the readme 'cause everyone has 'em and I would've found it useful. Language totally stolen from http://matplotlib.org/devel/gitwash/development_workflow.htm and just added a link to the dev workflow and making patch pages

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -47,3 +47,15 @@ The first is a good starting point for general questions and discussions.
...
+
+Contribute
+==========
+You've discovered a bug or something else you want to change - excellent!
+
+You've worked out a way to fix it – even better!
+
+You want to tell us about it – best of all!
+
+The easiest way to contribute is through github; if interested in doing so, browse the `development workflow <http://matplotlib.org/devel/gitwash/development_workflow.html>`_ guide.
```
Comment (QuLogic) [inline README.rst]: GitHub

File: README.rst [hunk 2]
```diff
@@ -47,3 +47,15 @@ The first is a good starting point for general questions and discussions.
...
+
+And if you don't want to use github, take a look at the guide to `making patches <http://matplotlib.org/devel/gitwash/patching.html>`_.
```
Comment (QuLogic) [inline README.rst]: GitHub; and we should probably fix the gitwash page, too.

File: README.rst [hunk 3]
```diff
@@ -47,3 +47,15 @@ The first is a good starting point for general questions and discussions.
...
+==========
+You've discovered a bug or something else you want to change - excellent!
+
+You've worked out a way to fix it – even better!
+
+You want to tell us about it – best of all!
+
+The easiest way to contribute is through Github; if interested in doing so, browse the `development workflow <http://matplotlib.org/devel/gitwash/development_workflow.html>`_ guide.
+
+And if you don't want to use Github, take a look at the guide to `making patches <http://matplotlib.org/devel/gitwash/patching.html>`_.
```
Comment (tacaswell) [inline README.rst]: I am not sure how this is not using github?

Comment (story645) [inline README.rst]: according to the [making patches ](http://matplotlib.org/devel/gitwash/patching.html#making-patches) docs, it's 'cause you go through every github step except the rebase/push madness,

> Then, send the generated patch files to the matplotlib mailing list — where we will thank you warmly.

File: README.rst [hunk 4]
```diff
@@ -47,3 +47,15 @@ The first is a good starting point for general questions and discussions.
...
+
+Contribute
+==========
+You've discovered a bug or something else you want to change - excellent!
+
+You've worked out a way to fix it – even better!
+
+You want to tell us about it – best of all!
+
+The easiest way to contribute is through Github; if interested in doing so, browse the `development workflow <http://matplotlib.org/devel/gitwash/development_workflow.html>`_ guide.
```
Comment (QuLogic) [inline README.rst]: GitHub, not Github.

Comment (story645) [inline README.rst]: 😶 I'll probably have to rebase this when it's all done.

**Comment by story645** [comment]
Ping...this is a a readme docfix so I'm gonna say the failure is travis being weird.

**Pull request merged**